recipes/menetools/1.0.4_1/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: menetools
+  version: "1.0.4_1"
+
+source:
+  fn: MeneTools-1.0.4.tar.gz
+  url: https://pypi.python.org/packages/6c/aa/3f85cb1ac67fdad4d4f59e39934a71bf3d0dbbe75027812c9a5b696ba766/MeneTools-1.0.4.tar.gz
+  md5: 28ed63db69417828b6b95eff141d984e
+
+requirements:
+  build:
+    - python >=3.5.4
```

Comment:
Can you remove the pinning here and skip the build in the build section for py27